### ISSUE TITLE
[Enhancement] Custom Pets

### ISSUE THREAD

**Issue by Volcan4436** [SmallCodee/SmallPets#26]
I would love top have a config file for you to add new Pets.

i would also love to have a GUI that lets me make a pet by selecting a skin from a website then selecting the attributes it gives and the level up attribute growth

**Comment by Harsh-7821** [comment]
yeah i config file idea is better than the original one and yeah InGame editor is better but that would need a lot of hard work so i guesssss first config update would be better and then we can wait for ingame update
